File: SDKs/api.video/1/python/src/sdk/models/shared/live_stream_assets.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json


@dataclass_json
@dataclass
class LiveStreamAssets:
    hls: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'hls' }})
    iframe: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'iframe' }})
    player: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'player' }})
    thumbnail: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'thumbnail' }})
    

File: SDKs/api.video/1/typescript/src/sdk/models/shared/livestreamassets.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class LiveStreamAssets extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=hls" })
  hls?: string;

  @SpeakeasyMetadata({ data: "json, name=iframe" })
  iframe?: string;

  @SpeakeasyMetadata({ data: "json, name=player" })
  player?: string;

  @SpeakeasyMetadata({ data: "json, name=thumbnail" })
  thumbnail?: string;
}
